### PR TITLE
[3.X] Dashboard pagination

### PR DESCRIPTION
**Pull request by haringsrob**
3.x port of https://github.com/area17/twill/pull/1889

Screenshot 2023-01-20 at 10 34 29

### CODE REVIEW

File: frontend/js/components/dashboard/activityFeed.vue [hunk 1]
```diff
@@ -71,14 +73,15 @@
       })
     },
     methods: {
+      getData(pageNumber) {
+        this.$http.get('admin?' + this.navFilters[this.navActive].slug + '=' + pageNumber).then(({data}) => {
```
Comment (haringsrob) [inline frontend/js/components/dashboard/activityFeed.vue:77]: @ifox do we have a helper to get the "admin" url?

File: config/dashboard.php [hunk 2]
```diff
@@ -24,5 +24,5 @@
         'login' => false,
         'logout' => false,
     ],
-    'auth_activity_causer' => 'users',
+    'auth_activity_causer' => 'users'
```
Comment (ifox) [inline config/dashboard.php:27]: Is our formatter config removing trailing commas? If you're not against them @haringsrob I would prefer to keep them. Diffs are more readable and it is less typing when adding a new line.

File: frontend/js/components/dashboard/activityFeed.vue (hunk 1 above)
Comment (ifox) [reply frontend/js/components/dashboard/activityFeed.vue:77]: Could we use the document location here?

**Review by ifox — CHANGES_REQUESTED** [review]

File: config/dashboard.php (hunk 2 above)
Comment (haringsrob) [reply config/dashboard.php:27]: Will add a standard.

File: frontend/js/components/dashboard/activityFeed.vue (hunk 1 above)
Comment (haringsrob) [reply frontend/js/components/dashboard/activityFeed.vue]: Decided to pass the base url from the backend to ensure it is correct. https://github.com/area17/twill/pull/2078/commits/92de5fc12f580c115eb3191ac85d6595a85cdbc3

File: config/dashboard.php (hunk 2 above)
Comment (haringsrob) [reply config/dashboard.php]: Not super relyable but I added https://github.com/squizlabs/PHP_CodeSniffer/issues/582

**Pull request merged**